WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectView.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+final class CardReaderSettingsConnectView: UIView {
+    @IBOutlet var contentView: UIView!
+    @IBOutlet weak var connectButton: UIButton!
+
+    var onPressedConnect: (() -> ())?
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        sharedInit()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        sharedInit()
+    }
+
+    private func sharedInit() {
+        Bundle.main.loadNibNamed("CardReaderSettingsConnectView", owner: self, options: nil)
```

Comment:
There are some helpers in UIView+Helpers that you might want to look into, that would help with this